app/models/donation.rb

```diff
@@ -44,21 +43,21 @@ def capture_transaction
   end
 
   def split_rules
-    organization_sr = PagarMe::SplitRule.new(organization_rule).create
-    city_sr = PagarMe::SplitRule.new(city_rule).create
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use a guard clause instead of wrapping the code inside a conditional expression.](https://www.codacy.com/app/jlcfigueira/hub-api/file/2610469914/issues/source?bid=3141163&fileBranchId=3282765#l46)